<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookViewPropertiesDialog</class>
 <widget class="QWidget" name="BookViewPropertiesDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>477</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>442</width>
    <height>477</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>442</width>
    <height>477</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Properties - Book View</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../Resources/Resources.qrc">
    <normaloff>:/icons/book_blue.png</normaloff>:/icons/book_blue.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <widget class="QTabWidget" name="m_optionsTab">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="Spire::BookViewLevelPropertiesWidget" name="m_levelsTab">
      <attribute name="title">
       <string>Levels</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout"/>
     </widget>
     <widget class="Spire::BookViewHighlightPropertiesWidget" name="m_highlightsTab">
      <attribute name="title">
       <string>Highlights</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3"/>
     </widget>
     <widget class="Spire::InteractionsWidget" name="m_interactionsTab">
      <attribute name="title">
       <string>Interactions</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2"/>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QGridLayout" name="m_buttonLayout">
     <item row="1" column="2">
      <widget class="QPushButton" name="m_resetDefaultButton">
       <property name="text">
        <string>Reset Default</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="m_saveAsDefaultButton">
       <property name="text">
        <string>Save As Default</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="m_loadDefaultButton">
       <property name="text">
        <string>Load Default</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="m_buttonCenterBottomSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <spacer name="m_buttonCenterTopSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="4">
      <widget class="QPushButton" name="m_applyButton">
       <property name="text">
        <string>Apply</string>
       </property>
      </widget>
     </item>
     <item row="0" column="5">
      <widget class="QPushButton" name="m_applyToAllButton">
       <property name="text">
        <string>Apply To All</string>
       </property>
      </widget>
     </item>
     <item row="1" column="4">
      <widget class="QPushButton" name="m_okButton">
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item row="1" column="5">
      <widget class="QPushButton" name="m_cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Spire::InteractionsWidget</class>
   <extends>QWidget</extends>
   <header>Spire/KeyBindings/InteractionsWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Spire::BookViewLevelPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>Spire/BookView/BookViewLevelPropertiesWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Spire::BookViewHighlightPropertiesWidget</class>
   <extends>QWidget</extends>
   <header>Spire/BookView/BookViewHighlightPropertiesWidget.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../Resources/Resources.qrc"/>
 </resources>
 <connections/>
</ui>
